db/migrate/20240117180824_latin1_to_utf8.rb

```diff
@@ -0,0 +1,15 @@
+class Latin1ToUtf8 < ActiveRecord::Migration[7.1]
+  def up
```

Comment:
1. Should I have separate migrations for Article and Sequence?
2. Is is possible to write this migration this without SQL, for instance using `:options`? I couldn't figure out how.

Reply:
Since this is a hot fix, I vote to do both migrations in one. I don't think it can be done without SQL, and from what i can tell from a quick search this seems like it would be valid SQL for any other db we might use.